src/ResultWriter/EnergyOutput.cpp

```diff
@@ -0,0 +1,142 @@
+#include "EnergyOutput.h"
+#include <Kernels/DynamicRupture.h>
+#include <Numerical_aux/Quadrature.h>
+#include <Parallel/MPI.h>
+
+real seissol::writer::computePlasticMoment(MeshReader const& i_meshReader,
+                                           seissol::initializers::LTSTree* i_ltsTree,
+                                           seissol::initializers::LTS* i_lts,
+                                           seissol::initializers::Lut* i_ltsLut) {
+  real plasticMoment = 0.0;
+  std::vector<Element> const& elements = i_meshReader.getElements();
+  std::vector<Vertex> const& vertices = i_meshReader.getVertices();
+
+#ifdef _OPENMP
+#pragma omp parallel for schedule(static) reduction(+ : plasticMoment)
+#endif
+  for (std::size_t meshId = 0; meshId < elements.size(); ++meshId) {
+    real* pstrainCell = i_ltsLut->lookup(i_lts->pstrain, meshId);
+    real volume = MeshTools::volume(elements[meshId], vertices);
+    CellMaterialData& material = i_ltsLut->lookup(i_lts->material, meshId);
+#ifdef USE_ANISOTROPIC
+    real mu = (material.local.c44 + material.local.c55 + material.local.c66) / 3.0;
+#else
+    real mu = material.local.mu;
+#endif
+    plasticMoment += mu * volume * pstrainCell[6 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS];
```

Comment:
Could you maybe add a comment explaining this datastructure. At this point in the code it comes a bit out of nothing and it would be great to have an explanation there.